tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -341,14 +361,6 @@ batch:
         variables:
           AWS_LC_CI_TARGET: "tests/ci/run_prefix_tests.sh"
 
-    - identifier: amazonlinux2_gcc7x_intel_sde_x86_64
```

Comment:
It makes sense that we needed the new version to test ASAN + SDE, what are your thoughts about keeping AL2 + SDE? Most internal teams will be running on AL2 and I like testing all CPUs with it. Granted if we have Ubuntu + SDE, I'm not sure we need AL2 + SDE. At that point are we testing the OS/system calls on various CPUs more than AWS-LC?

Reply:
Sure we could keep the AL2 dimension and use Ubuntu for SDE+ASAN instead. I like the idea of testing both platforms. I was initially trying to upgrade this to AL2023, but chose ubuntu instead since AL2023 was [phasing out 32-bit runtime support](https://docs.aws.amazon.com/linux/al2023/ug/compare-with-al2.html#i686) (which `sde` needs).